packages/resizin/README.md

```diff
@@ -2,112 +2,233 @@
+
     
-### Upload
-* Upload in browser (get file using jQuery)
-    ```javascript
-        $('#fileinput').live('change', function(){ 
-          	var files = $('#fileinput').prop('files');
-            var promise = bucketUploader( 
-                id = "File name", 
-                files[0]
-            );
-        });
-    ```
+Examples from the wild:
+
+#### Upload in client (get file using jQuery)
+
+Assume you have a file input in your page
+
+```HTML
+<input type="file" id="fileinput" />
+```
+
+then
+
+```js
+import { upload } from 'resizin';
+
+$('#fileinput').live('change', function(){ 
+    var files = $('#fileinput').prop('files');
     
-    ```HTML
-        <input type="file" id="fileinput" />
-    ```
-
-* Upload in Node.js
-    ```javascript
-        const fs = require('fs');
-
-        const file = fs.createReadStream(__dirname + '/myfile.png');
-        var promise = bucketUploader(
-            'imageid', 
-            file
-        );
-    ```
-
-## Build URL component
-
-* Package import
-    ```javascript
-        ES5: const buildSource = require("ackee-image-server").buildSource;
-        ES6: import { buildSource } from "ackee-image-server";
-    ```
-
-* Bucket initialization
-    ```javascript
-        const bucketBuilder = buildSource.bind(
-            null, 
-            url = 'https://img.ack.ee', 
-            bucket_alias = "test"
-        );
-    ```
-
-* Building URL
-    ```javascript
-        var url = bucketBuilder(
-            id = 'pepus', 
-            {
-                width: 250, 
-                left: '20', 
-                filter: 'sepia', 
-                rotate: 90, 
-                border: [10, 20]
-            }
-        )
-    ```
-
-* Result
-
-Built URL in a HTML element `<img>`
-
-![alt tag](https://img.ack.ee/test/image/w_250-x_20-f_sepia-r_90-b_10_20/pepus)
-
-* Modifiers
-List of available modifiers.
-
-| Modifier   |      Shortcut      |  Available options |
-|----------|:-------------:|------:|
-| width | w | |
-| height | h | |
-| filter | f | sepia, grayscale, sharpen, blur, negative, edge, gauss |
-| size | s | |
-| gravity | g | north, south, east, center, west, northeast, southeast, southwest, face |
-| crop | c | fill, fit, pad, scale, cut, face |
-| left | l | |
-| top | t | |
-| rotate | r | 90, 180, 270, 360 |
-| border | b | [top, right, bottom, left] |
-| backgroundColor | bg | #hexColor, rgb() |
-| quality | q | |
-| upscale | u | |
-
-Modifiers usage demo <a href="https://imageserver.ack.ee/" target="_blank">here</a>.
+    upload(
+        'https://api.resizin.com',
+        config.RESIZIN_API_KEY,
+        'imageid', 
+        files[0]
+    ).then(() => {
+        ...      
+    });
+});
+```
+
+#### Upload in Node.js
+```js
+const resizin = require('resizin');
+const fs = require('fs');
+const config = require('../config');
+
+const file = fs.createReadStream(__dirname + '/myfile.png');
+var promise = resizin.upload(
```

Comment:
is it functional? have you tried it for real? 
cause I think our imageserver had some troubles with this

Reply:
what is the response from server after uploading file?

Reply:
No I didn't, it's taken from the original package. I'm working on `node-resizin` package in different branch now, so I will check it there. This MR is more about `buildSource` :)